inc/ThirdParty/Plugins/SEO/SEOPress.php

```diff
@@ -27,9 +27,14 @@ public function __construct( Options_Data $option ) {
 	 * Subscribed events.
 	 */
 	public static function get_subscribed_events() {
-		if ( ! function_exists( 'seopress_get_toggle_xml_sitemap_option' ) || 1 !== (int) seopress_get_toggle_xml_sitemap_option() ) {
+		if ( ! function_exists( 'seopress_get_toggle_option' ) || '1' !== seopress_get_toggle_option( 'xml-sitemap' ) ) {
```

Comment:
What happens for someone with an older version of the SEOPress and this version of WPR? Does this function exist in the older versions?